mini-wallet.md

```diff
@@ -113,25 +109,43 @@ Options:
   -d, --test-debug-api BOOLEAN    Run tests for debug APIs.  [default: False]
   -v, --verbose                   Enable verbose log output.  [default: False]
   --help                          Show this message and exit.
-```
+
+  ```
+
 
 ### How it works
 
-1. The test suite is located in the `diem.testing.suites` package and includes the pytest file `conftest.py`.
-2. `dmw test` launches a pytest runner to run tests.
-3. The `conftest.py` starts a stub MiniWallet as a counterparty service for testing payment with the target server specified by the `--target` option.
+The `diem.testing.suites` package contains the Diem MiniWallet Test Suite and includes the pytest file `conftest.py`.
+
+Launch a pytest runner to run tests using:
+
+`dmv test`
+
+The `conftest.py` starts a stub MiniWallet as a counterparty service for testing payment with the target server specified by the --target option here.
+
+
+### Work with your local test network
 
+The Diem MiniWallet server connects to the Diem testnet by default. However, you can use the MiniWallet to test your wallet application in your own local test network as well.
 
-### Working with a local testnet
+To run your local test network:
+1. Download and install Docker and Docker Compose. 
+2. Download the Diem testnet docker compose configuration file.
```

Comment:
`https://github.com/diem/diem/blob/master/docker/compose/validator-testnet/docker-compose.yaml` is important